packages/block-editor/src/components/block-toolbar/unified.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { Popover } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { store as blockEditorStore } from '../../store';
+import BlockContextualToolbar from '../block-list/block-contextual-toolbar';
+
+export default function UnifiedBlockToolbar() {
+	const hasFixedToolbar = useSelect(
+		( select ) => select( blockEditorStore ).getSettings().hasFixedToolbar,
+		[]
+	);
+
+	if ( hasFixedToolbar ) {
+		return (
+			<>
+				<Popover.Slot name="block-toolbar" />
+				<BlockContextualToolbar hideDragHandle className="is-fixed" />
```

Comment:
Minor but I think it'd be preferable to make this an `isFixed` prop. 

Potentially, `hideDragHandle` won't be needed then, as I imagine it's only true when `isFixed` is also true.

Reply:
Sure, sounds good